<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>953</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="symbol_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Symbol</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="symbol_lineedit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="intraday_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Intraday</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">interval_group</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="daily_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Daily</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">interval_group</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="weekly_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Weekly</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">interval_group</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="monthly_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Monthly</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">interval_group</string>
    </attribute>
   </widget>
   <widget class="QComboBox" name="interval_options">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>1 Min</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5 Min</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15 Min</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>30 Min</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>60 Min</string>
     </property>
    </item>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>951</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGraphicsView" name="stock_plot">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>651</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="submit_button">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>90</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="high">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>High</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="low">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Low</string>
    </property>
   </widget>
   <widget class="QLabel" name="info">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>200</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>info</string>
    </property>
   </widget>
   <zorder>symbol_label</zorder>
   <zorder>intraday_button</zorder>
   <zorder>daily_button</zorder>
   <zorder>weekly_button</zorder>
   <zorder>monthly_button</zorder>
   <zorder>interval_options</zorder>
   <zorder>line</zorder>
   <zorder>stock_plot</zorder>
   <zorder>submit_button</zorder>
   <zorder>high</zorder>
   <zorder>Low</zorder>
   <zorder>info</zorder>
   <zorder>symbol_lineedit</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>953</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="interval_group"/>
 </buttongroups>
</ui>
